database/models/files.db.php

```diff
@@ -0,0 +1,44 @@
+<?php
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\GeneratedValue;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\ManyToOne;
+use Doctrine\ORM\Mapping\Table;
+
+#[Entity, Table(name: 'shared_documents')]
+class SharedFile
+{
+    #[Id, Column, GeneratedValue]
+    public int|null $id = null;
+
+    #[Column]
+    public string $path = "";
+
+    #[Column]
+    public DateTime $date;
+
+    #[Column]
+    public string $mime = "";
+
+    #[Column]
+    public bool $is_public = true;
+
+    #[ManyToOne]
+    public Race|null $race = null;
```

Comment:
Keep in mind you need to add the other side of the relation to `Event` and `Race` entities so that you can just join them easily. You can do this in another PR, try to take inspiration from the relation between `Event` and `Race`